multimodal/src/autogluon/multimodal/utils/model.py

```diff
@@ -523,3 +524,37 @@ def modify_duplicate_model_names(
 
 def list_timm_models(pretrained=True):
     return timm.list_models(pretrained=pretrained)
+
+
+def _filter_timm_pretrained_cfg(cfg, remove_source=False, remove_null=True):
+    filtered_cfg = {}
+    keep_null = {"pool_size", "first_conv", "classifier"}  # always keep these keys, even if none
+    for k, v in cfg.items():
+        if remove_source and k in {"url", "file", "hf_hub_id", "hf_hub_id", "hf_hub_filename", "source"}:
+            continue
+        if remove_null and v is None and k not in keep_null:
+            continue
+        filtered_cfg[k] = v
+    return filtered_cfg
+
+
+def save_timm_config(
+    model: TimmAutoModelForImagePrediction,
+    config_path: str,
+):
+    hf_config = {}
```

Comment:
The name `hf_config` looks more like a config for huggingface models.

Reply:
Updated the name.